### ISSUE TITLE
pod try fails

### ISSUE THREAD

**Issue by jakubknejzlik** [thefuntasty/TFTableDescriptor#4]
when running pod install, the script returns following error: 

```
Fetching podspec for `TFTableDescriptor` from `../`
[!] Unable to satisfy the following requirements:

- `TFTableDescriptor (from `../`)` required by `Podfile`
- `TFTableDescriptor (from `../`)` required by `Podfile`
- `TFTableDescriptor (= 0.1.0)` required by `Podfile.lock`
```

**Comment by jakubknejzlik** [comment]
got it...when publishing new version the podfile.lock is sticks with previous version of pod, which fails to work with newer version. The solution could be to run `pod update` before publishing the new pod version or move Example/Podfile.lock to .gitignore

**Comment by jakubknejzlik** [comment]
commit `a6437f79ab3079cdf85c71d8a316697117ad10c0` contains updated Podfile.lock (checkout the version change)

**Comment by jakubknejzlik** [comment]
Also current version 0.3.0 can be fixed by running `pod update` in Example folder (after `pod try` is completed)

**Comment by kafejo** [comment]
Fixed in [1.0.0 release](https://github.com/thefuntasty/TFTableDescriptor/releases/tag/1.0.0).
